src/components/exercise-table/Exercise-table.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+
+// Define the Set type
+interface Set {
+  reps: number;
+  weight: number;
+}
+
+// Define the Exercise type
+interface Exercise {
```

Comment:
You could extend this interface (since we defined `Exercise` in `/lib/exercise.ts` 
This looks like it's an exercise entry specific to the table? 

```suggestion
interface ExerciseEntry extends Exercise { 
  sets: Set[];
}
```